config/environments/production.rb

```diff
@@ -26,6 +26,9 @@
   config.assets.js_compressor = :uglifier
   # config.assets.css_compressor = :sass
 
+  # Set a css_compressor so sassc-rails does not overwrite the compressor when running in production
+  config.assets.css_compressor = nil
```

Comment:
We're effectively doing this in all environments now (since it's implied by default for development). Is there somewhere we can specify it only once? `config/application.rb`?